tests/testsuite/test_acceptance_with_browsers.py

```diff
@@ -104,20 +104,20 @@ def browser(request):
         yield browser
 
 
-@pytest.mark.parametrize('browser', ['firefox'], indirect=True)
+@pytest.mark.parametrize('browser', [BROWSERS[0]], indirect=True)
```

Comment:
I'd rather see `firefox` here.

Reply:
Yeah but I don't care which browser it is. And I kept adding/removing browsers chrome/firefox on the list until the UI guys made the menu bar work for firefox too.

Main issue is that we have to run that specific test only once, because it changes state. I wish there was a way to test the initial welcome wizard with both browsers, then test the rest of the things in a separate fashion.